R/Table.R

```diff
@@ -62,6 +71,9 @@ odbc_write_table <-
         )
       rs <- OdbcResult(conn, sql)
 
+      if (!is.null(fieldDetails) && nrow(fieldDetails))
+        result_describe_parameters(rs@ptr, fieldDetails)
```

Comment:
Can you put braces around this conditional?

Reply:
Will do - thanks.